api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -154,6 +154,10 @@ type DRPlacementControlSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf", message="pvcSelector is immutable"
 	PVCSelector metav1.LabelSelector `json:"pvcSelector"`
 
+	// Label selector to identify the cephfs PVCs that are in a consistency group
+	// nil means no consistency group. metav1.LabelSelector{} means all pvcs in a consistency group
+	CephFSConsistencyGroupSelector *metav1.LabelSelector `json:"cephFSCGSelector,omitempty"`
```

Comment:
remove this selector, and add new label in the cephfs pvc. then use a label selector to select the cepfhfs pvc